api/messaging/sms/sms_message_request.py

```diff
@@ -132,22 +132,26 @@ def schedule(self):
         requirements.
         """
 
-        if Validate.message(self.message) and Validate.number(self.to) and \
-                Validate.date_and_time_validation(self.datetime, self.format):
-            correct_time = Validate.date_and_time_validation(self.datetime, self.format)
-            url = '{}&method={}&message={}&to={}&sender={}&time={}'.format(BASEURL, SMSMessageRequest.SMS, self.message,
-                                                                           self.to, SENDERID, correct_time)
-            if self.dlr_url:
-                url += '&dlrurl={}'.format(self.dlr_url)
-            if self.custom:
-                url += '&custom={}'.format(self.custom)
-            if self.unicode:
-                url += '&unicode={}'.format(self.unicode)
-            if self.flash:
-                url += '&flash={}'.format(self.flash)
-            response = Klient(url).response()
-            sms_message_response = SMSMessageResponse(response=response)
-            return sms_message_response
+        if (
+            not Validate.message(self.message)
+            or not Validate.number(self.to)
+            or not Validate.date_and_time_validation(self.datetime, self.format)
+        ):
+            return
+
+        correct_time = Validate.date_and_time_validation(self.datetime, self.format)
+        url = '{}&method={}&message={}&to={}&sender={}&time={}'.format(BASEURL, SMSMessageRequest.SMS, self.message,
+                                                                       self.to, SENDERID, correct_time)
+        if self.dlr_url:
+            url += '&dlrurl={}'.format(self.dlr_url)
+        if self.custom:
+            url += '&custom={}'.format(self.custom)
+        if self.unicode:
+            url += '&unicode={}'.format(self.unicode)
+        if self.flash:
+            url += '&flash={}'.format(self.flash)
+        response = Klient(url).response()
+        return SMSMessageResponse(response=response)
```

Comment:
Function `SMSMessageRequest.schedule` refactored with the following changes:

- Add guard clause ([last-if-guard](https://docs.sourcery.ai/refactorings/last-if-guard/))
- Inline variable that is immediately returned ([inline-immediately-returned-variable](https://docs.sourcery.ai/refactorings/inline-immediately-returned-variable/))